src/components/Sidebar/Sidebar.tsx

```diff
@@ -112,7 +114,8 @@ const actionBarCss = css`
 
 function ActionBar() {
   const [opHistory, setOpHistory] = useModelState('spec_history');
-  const spec = useModelState('graph_spec')[0];
+  const [spec, setSpec] = useModelState('graph_spec');
+  const [selectedData] = useModelState('selected_data');
```

Comment:
Can we do this if we just want the first element?

Reply:
Sorry I don't know what you are asking here

Reply:
`const [selectedData] = useModelState('selected_data');` useModelState returns an array with two elements. In this case, selectedData would have the first element right?

Reply:
NVM

Reply:
Yeah it's roughly equivalent to 'const selectedData = useModelState('selected_data')[0]'